File: app/src/main/java/ai/elimu/launcher/MainActivity.java
package ai.elimu.launcher;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import ai.elimu.launcher.ui.HomeScreensActivity;
import timber.log.Timber;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

            Intent intent = new Intent(getApplicationContext(), HomeScreensActivity.class);
            startActivity(intent);

        finish();
    }

    @Override
    public void onBackPressed() {
        Timber.i("onBackPressed");

        // Do nothing
    }
}

File: app/src/main/java/ai/elimu/launcher/MainActivity.kt
package ai.elimu.launcher

import ai.elimu.launcher.ui.HomeScreensActivity
import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import timber.log.Timber

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(applicationContext, HomeScreensActivity::class.java)
        startActivity(intent)

        finish()
    }

    @SuppressLint("MissingSuperCall")
    override fun onBackPressed() {
        Timber.i("onBackPressed")

        // Do nothing
    }
}
